src/Microsoft.ComponentDetection.Detectors/pip/PythonVersionUtilities.cs

```diff
@@ -9,8 +9,8 @@ public static class PythonVersionUtilities
         /// <summary>
         /// Determine if the version is valid for all specs.
         /// </summary>
-        /// <param name="version">Version.</param>
-        /// <param name="specs">Version specifications.</param>
+        /// <param name="version">version.</param>
+        /// <param name="specs">version specifications.</param>
```

Comment:
This looks like an unrelated change

Reply:
since name of the parameter changed, we should also change the documentation ?